src/xclim/indices/_threshold.py

```diff
@@ -1914,6 +1915,10 @@ def snowfall_intensity(
     return snow_int
 
 
+@deprecated(
+    "`heat_wave_index` will be deprecated in a future version. "
+    "Use `hot_spell_total_length` instead"
+)
```

Comment:
Check out the signature of `deprecated()` ;)

Reply:
Yes I had not checked it, I just want to say I had planned to check it on my own :P ... but I should not throw in the dark like this.